plugins/woocommerce/includes/class-wc-validation.php

```diff
@@ -70,6 +70,8 @@ public static function is_postcode( $postcode, $country ) {
 				$valid = (bool) preg_match( '/^(DK-)?([1-24-9]\d{3}|3[0-8]\d{2})$/', $postcode );
 				break;
 			case 'ES':
+			case 'FI':
+			case 'EE':			
```

Comment:
```suggestion
			case 'EE':
```

Reply:
Apparently there were. Thanks for resolving